### DISCUSSION TITLE
what functions in Signac take advantage of the future framework and will be parallelized if the current plan is set appropriately?

### DISCUSSION THREAD

**Discussion by felixm3** [stuart-lab/signac#1491] [Q&A] [answered]
Seurat lists functions that take advantage of the future framework and will be parallelized if the current plan is set appropriately.

https://satijalab.org/seurat/archive/v3.1/future_vignette.html

<img width="878" alt="image" src="https://github.com/stuart-lab/signac/assets/43228120/2bc06e72-bb4a-4965-9b76-acd4f87f16b9">

Is there a similar list somewhere for Signac functions?

**Comment by timoast** [comment, marked as answer]
- Quantification (FeatureMatrix and related functions)
- TSS enrichment
- Footprinting
- LinkPeaks

We can add this information to the [vignette](https://stuartlab.org/signac/articles/future)
